README.md

```diff
@@ -9,7 +9,7 @@ The Beacon SDK requires iOS 9.0+.
 
 ### CocoaPods
 
-Beacon is available through [CocoaPods][1]. To install it, simply add the following line to your `Podfile`:
+Beacon is available through [CocoaPods][1]. To install it, add the following line to your `Podfile`:
```

Comment:
:+1: I still have a bad habit of adding "simply" to stuff, which I think does more harm than good.

Reply:
The grammarly hint is always, sound more confident, remove this word.